azext_iot/iothub/commands_device_messaging.py

```diff
@@ -183,6 +183,7 @@ def iot_simulate_device(
     hub_name: Optional[str] = None,
     resource_group_name: Optional[str] = None,
     login: Optional[str] = None,
+    model_id: Optional[str] = None,
```

Comment:
should this be added for send d2c message too?

Reply:
Debatable, but I don't believe it is necessary for this PR. Device simulation supports configuring an initial device state which model Id fits well. Where-as send-d2c-message are one-off messages from a prior created (or configured) device identity. The user is able to set a model id via simulate, then use send-d2c-message against the device with prior-set model Id.

Reply:
what do you mean by settable? If a user runs simulate with model A, would they need to input the model id on the next run again?

Reply:
Added `--model-id` to send-d2c-message as well.